jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/DataSetUtils.java

```diff
@@ -57,9 +57,17 @@ public static Boolean getColumnBooleanValue(DataSet currentDataSet,
     }
 
     public static Integer getColumnIntValue(DataSet currentDataSet,
-                                        String columnId,
-                                        int index) {
+                                            String columnId,
+                                            int index) {
         Object value = currentDataSet.getValueAt(index, columnId);
         return value != null ? Integer.parseInt(value.toString()) : null;
     }
+    
+    public static Short getColumnShortValue(DataSet currentDataSet,
+                                            String columnId,
+                                            int index) {
+        Object value = currentDataSet.getValueAt(index, columnId);
+        return value != null ? Short.parseShort(value.toString()) : null;
+    }
+
```

Comment:
Are we aiming at memory saving here with introduction of this method over existing _getColumnIntValue_?
If not, I suggest reusing _getColumnIntValue_ and not overcrowd the utils class.